docs/breaking.md

```diff
@@ -79,19 +91,19 @@ The relevant HTTP 5xx status codes will be used in the situation where a server
 
 ### Changes to Routes
 
-- [Get Test Metadata](https://circleci.com/docs/api/v2/#get-test-metadata) endpoint will move from `/project/{project-slug}/{job-number}/tests` to `/projectS/{project-slug}/jobs/{job-number}/tests`. Note the addition of `jobs/` to the route
-- References to `project/` will be pluralized to `projectS/` e.g.
-  - `/project/{project-slug}` -> `/projects/{project-slug}`
-  - `/project/{project-slug}/job/{job-number}` -> `/projects/{project-slug}/jobs/{job-number}`
-- References to `pipeline/` will be pluralized to `pipelineS/` e.g.
-  - `/pipeline/{pipeline-id}` -> `pipelineS/{pipeline-id}`
-  - `/pipeline/{pipeline-id}/config` -> `/pipelineS/{pipeline-id}/config`
-- References to `job/` will be pluralized to `jobS/` e.g.
-  - `/project/{project-slug}/job/{job-number}` -> `/projectS/{project-slug}/jobS/{job-number}`
-- References to `workflow/` will be pluralized to `workflowS/` e.g.
-  - `/workflow/{id}` -> `/workflowS/{id}`
+- [Get Test Metadata](https://circleci.com/docs/api/v2/#get-test-metadata) endpoint will move from `/project/{project-slug}/{job-number}/tests` to `/project/{project-slug}/job/{job-number}/tests`. Note the addition of `job/` to the route
+- [Get a workflow's jobs](https://circleci.com/docs/api/v2/#get-a-workflow-39-s-jobs) `/workflow/{id}/jobs` will be singular and now live under `/workflow/{id}/job`
+
 
 ### Paginated Responses
 
 - [Get an environment variable](https://circleci.com/docs/api/v2/#get-an-environment-variable) response will be paginated
 - [Get all checkout keys](https://circleci.com/docs/api/v2/#get-all-checkout-keys) response will be paginated
+
+### Changes to Responses
+
+#### Get a Pipeline
+A pipeline could potentially have an unbounded set of workflows associated with it. For that reason we are removing `workflow` from the [get a pipeline](https://circleci.com/docs/api/v2/#get-a-pipeline) response and introducing two new endpoints:
+
+1. Get a pipeline's workflow `GET /pipeline/{pipeline-id}/workflow`
```

Comment:
Should read: 
1. Get a pipeline's workflows ...

Reply:
NO PLURALS :P